operator/controller/src/test/java/io/apicurio/registry/operator/unit/CorsTest.java

```diff
@@ -0,0 +1,98 @@
+package io.apicurio.registry.operator.unit;
+
+import io.apicurio.registry.operator.EnvironmentVariables;
+import io.apicurio.registry.operator.OperatorException;
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.apicurio.registry.operator.feat.Cors;
+import io.apicurio.registry.operator.utils.Mapper;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.nio.charset.Charset;
+import java.util.LinkedHashMap;
+import java.util.Set;
+
+public class CorsTest {
+
+    private static final String DEFAULT = """
+            apiVersion: registry.apicur.io/v1
+            kind: ApicurioRegistry3
+            metadata:
+              name: simple
+            spec: {}
+            """;
+
+    private static final String WITH_INGRESS = """
+            apiVersion: registry.apicur.io/v1
+            kind: ApicurioRegistry3
+            metadata:
+              name: simple
+            spec:
+              app:
+                ingress:
+                  host: simple-app.apps.cluster.example
+              ui:
+                ingress:
+                  host: simple-ui.apps.cluster.example
+            """;
+
+    private static final String WITH_ENV_VAR = """
+            apiVersion: registry.apicur.io/v1
+            kind: ApicurioRegistry3
+            metadata:
+              name: simple
+            spec:
+              app:
+                env:
+                  - name: QUARKUS_HTTP_CORS_ORIGINS
+                    value: https://ui.example.org
+            """;
+
+    private static final String WITH_ENV_VAR_AND_INGRESS = """
```

Comment:
I think we should use the files in `operator/controller/src/test/resources/k8s/examples` as much as possible, even if they are simple (or slightly edited in the test). This would provide resources we can use in the docs and where we can point users to. The examples will end up in the "install examples" release artifact.

(BTW similarly I have an aspiration to eventually do the same thing for the registry examples and run them as tests).

Reply:
I was having trouble getting those resources to load in a unit test vs an IT test (the context classloader is different).  I didn't want to duplicate the resource loading, but I *can* of course.  I'll go ahead and move these into `k8s/exmaples`.